python-main.js

```diff
@@ -342,20 +349,6 @@ function web_editor(config) {
             EDITOR.setCode(config.translate.code.start);
         }
         EDITOR.ACE.gotoLine(EDITOR.ACE.session.getLength());
-        // If configured as experimental update editor background to indicate it
-        if(config.flags.experimental) {
-            EDITOR.ACE.renderer.scroller.style.backgroundImage = "url('static/img/experimental.png')";
-            $("#known-issues").removeClass('hidden');
-        }
-        // Configure the zoom related buttons.
-        $("#zoom-in").click(function (e) {
-            e.stopPropagation();
-            zoomIn();
-        });
-        $("#zoom-out").click(function (e) {
-            e.stopPropagation();
-            zoomOut();
-        });
```

Comment:
Button click listeners also moved to setupButtons(), where all the other buttons are set.